WordPress/create-block-theme
# Persist copyright info on new theme creation

This PR ensures that existing third-party credits/copyright are maintained when a theme is cloned, or a child theme is created. 

At the moment, when a theme is cloned, the readme.txt included in the new theme is not based on the current theme, it's a completely new readme.txt file. This PR copies any existing copyright information into the new readme.txt file, to help ensure all copyright info is maintained. 

A default copyright section is used for blank themes, as they aren't based on the active theme. I've also hidden the image credits input on the 'Create blank theme' option, as this option creates an empty theme and doesn't include any assets, so it doesn't make sense to include any image credits.

Closes https://github.com/WordPress/create-block-theme/issues/381.

### Testing Instructions
For testing a theme with new font assets:
1. Add a Google font to the active theme (to generate some example font license info)
2. Ensure that the font license info has been added to the current theme's readme.txt
3. Go to either the Create Block Theme wp-admin page or the Site Editor sidebar panel
4. Create a clone or a child theme
5. The font license info should be in the new theme's readme.txt file

For testing a theme with existing third-party credits:
1. Activate a theme with third-party credits, e.g. Pixl
2. Go to either the Create Block Theme wp-admin page or the Site Editor sidebar panel
4. Create a clone or a child theme
5. The third-party copyright info should be in the new theme's readme.txt file

Please also try testing the creation of a blank theme, as this should include a new default copyright notice.
